packages/sdk/src/evm/lib/static-batch-rpc.ts

```diff
@@ -66,6 +66,10 @@ export class StaticJsonRpcBatchProvider extends providers.StaticJsonRpcProvider
         // on whether it was a success or error
         batch.forEach((inflightRequest_, index) => {
           const payload = result[index];
+          // if there is no payload, reject the request
+          if (!payload) {
```

Comment:
Did we see this crash somewhere? 
Do you think this error is because we got less results than items in the batch or because we are getting undefined for some of them? If it's the first case then all results after the first failure will be assigned to incorrect requests.

Reply:
yep, this crashed some folks in dashboard and now that we finally have sourcemaps in sentry we caught it :)